Classes/Telemetry/BITEventData.m

```diff
@@ -5,13 +5,17 @@ @implementation BITEventData
 @synthesize envelopeTypeName = _envelopeTypeName;
 @synthesize dataTypeName = _dataTypeName;
 @synthesize version = _version;
+@synthesize properties = _properties;
+@synthesize measurements = _measurements;
 
 /// Initializes a new instance of the class.
 - (instancetype)init {
   if (self = [super init]) {
     _envelopeTypeName = @"Microsoft.ApplicationInsights.Event";
     _dataTypeName = @"EventData";
     _version = @2;
+    _properties = [NSDictionary new];
```

Comment:
If we create a new dict instance here, it will always show up in the request body unless the user is not explicitly setting it to `nil`. I'd probably do that in the `setProperty` method.